src/test/java/uk/gov/hmcts/reform/idam/health/probe/ScheduledHealthProbeIndicatorTest.java

```diff
@@ -105,4 +105,12 @@ public void testIsOkay_ignoreDown() {
         assertThat(ignoringScheduledHealthProbe.isOkay(), is(true));
         verify(healthProbe, times(2)).probe();
     }
+
+    @Test
+    public void testIsOkay_alwaysTrueWhenIgnoreProbe() {
```

Comment:
This is a good test but it does not cover the scenario that was failing in our latest incident: should be `testIsOkay_successWhenNotFreshAndIgnoreDown`

Reply:
The problem with that test is that when a Probe is IGNORE and its probe check is false, at the moment, we are not updating the status to DOWN (we are just printing it). Is that correct?

Reply:
Yeah, that's true - sorry I've misspelled the name of the test, should be `testIsOkay_successWhenNotFreshAndIgnore` (isOkay() returns `true` even when `LocalDateTime.now(clock).isBefore(statusDateTime.plus(freshnessInterval, ChronoUnit.MILLIS));`